src/server/utils/validations.js

```diff
@@ -49,6 +49,14 @@ const validations = {
     }
     return validatedString;
   },
+
+  validateName: (data, count, nullable) => {
```

Comment:
Maybe calling it `nameToUpperCase`, just to be super explicit about what this function does?